game/engine/persistent_server.py

```
@@ -0,0 +1,142 @@
+        try:
+            os.unlink(port_path)
+            os.unlink(pid_path)
+        except OSError:
+            pass
+        sys.exit(1)
+
+    inactivity_timeout = 300.0 # 5 minutes of no activity
+    try:
+        listener._listener._socket.settimeout(inactivity_timeout)
+    except Exception:
+        pass
```

Ruff (0.15.20)

[warning] Use `contextlib.suppress(Exception)` instead of `try`-`except`-`pass`

Replace `try`-`except`-`pass` with `with contextlib.suppress(Exception): ...`

(SIM105)